server/src/server.ts

```diff
@@ -140,12 +265,16 @@ connection.onDidChangeConfiguration((change) => {
 });
 
 function validateTextDocument(textDocument: TextDocument): void {
```

Comment:
If you asked me out of the blue, I would have said that “validation” was part of “parsing” rather than the opposite. Consider adding doc for dumb people like me that have it backwards.

Reply:
I can rename it and document it. OK.